x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/session/Configuration.java

```diff
@@ -163,9 +174,17 @@ public String query() {
      * Note: Currently, it returns {@link System#currentTimeMillis()}, but this value will be serialized between nodes.
      */
     public long absoluteStartedTimeInMillis() {
+        // MP TODO: I'm confused - Why is this not a fixed value taken at the start of the query processing?
```

Comment:
Yikes. I'll bet this is old. It should totally be serialized too. *looks* like a bug to me.

Reply:
But not as part of this PR.

Reply:
I opened https://github.com/elastic/elasticsearch/issues/113709

Reply:
I think we could just use `queryStartTimeMillis` here. but not in this PR because I have no idea if it'd do something unexpected.

Reply:
Yeah, these two fields/methods feel redundant but this PR has so much going on I didn't want to futz with it here since the pre-existing case is unclear to me. Thanks for creating a follow-on ticket.